build-logic-settings/build-environment/src/main/kotlin/gradlebuild/basics/BuildEnvironmentService.kt

```diff
@@ -29,6 +30,14 @@ abstract class BuildEnvironmentService : BuildService<BuildEnvironmentService.Pa
 
     interface Parameters : BuildServiceParameters {
         val rootProjectDir: DirectoryProperty
+        val rootProjectBuildDir: DirectoryProperty
+        val artifactoryUserName: Property<String>
+        val artifactoryPassword: Property<String>
+        val testVersions: Property<String>
+        val integtestAgentAllowed: Property<String>
+        val integtestDebug: Property<String>
+        val integtestLauncherDebug: Property<String>
+        val integtestVerbose: Property<String>
```

Comment:
❓ Can we trim this list of parameters now that we use `providers.gradleProperty` for most things?

Reply:
Good catch, removed these parameters